css/style.css

```diff
@@ -1309,9 +1333,20 @@ body#Home #content > .intro .download
 
 body#Home #content > .intro .download .btn
 {
-    margin-bottom: 0.3em;
+    margin: 0 0.3em 0.3em 0.3em;
+}
+
+body#Home #content > .intro .download .btn.action
+{
+    border-color: #98312A;
 }
 
+body#Home #content > .intro .download .btn.action:hover
+{
+    border-color: #943029;
+}
+
+
```

Comment:
It's a bit problematic b/c of synchronizing the colors, but at least separating out all that `body#Home` stuff would make sense.